clayground.py

```diff
@@ -0,0 +1,130 @@
+import lancedb
+import matplotlib.pyplot as plt
+import rasterio as rio
+import streamlit as st
+from rasterio.plot import show
+
+st.set_page_config(layout="wide")
+
+
+# Get preferrred chips
+def get_unique_chips(tbl):
+    chips = [
+        {"tile": "17MNP", "idx": "0271", "year": 2023},
+        {"tile": "19HGU", "idx": "0033", "year": 2018},
+        {"tile": "33NVB", "idx": "0393", "year": 2020},
+        {"tile": "21JVJ", "idx": "0100", "year": 2020},
+        {"tile": "34KHD", "idx": "0080", "year": 2018},
+        {"tile": "19JCF", "idx": "0215", "year": 2023},
+        {"tile": "20HMK", "idx": "0100", "year": 2020},
+        {"tile": "37MFT", "idx": "0313", "year": 2023},
+        {"tile": "49KHR", "idx": "0020", "year": 2017},
+        {"tile": "55LBC", "idx": "0075", "year": 2022},
+    ]
+
+    filter = " OR ".join(
+        [
+            f"(tile == '{chip['tile']}' AND idx == '{chip['idx']}') AND year == {chip['year']}"
+            for chip in chips
+        ]
+    )
+    result = tbl.search().where(filter, prefilter=True).to_pandas()
+    return result
+
+
+# Load embeddings
+@st.cache_resource()
+def connect_to_database():
+    db = lancedb.connect("nbs/embeddings")
+    tbl = db.open_table("clay-v001")
```

Comment:
Will need to document how the `nbs/embeddings' LanceDB vector database is created. Can do this afterwards.